src/app/components/registaration/registration.component.scss

```diff
@@ -0,0 +1,31 @@
+$border-solid-1-red: 1px solid red;
```

Comment:
You use this style in different files, it will be good to move this and other reusable values in a separate file, and then import